common/src/main/java/io/seata/common/util/StringUtils.java

```diff
@@ -373,4 +372,22 @@ public static boolean hasUpperCase(String str) {
         return false;
     }
 
+    public static boolean hasText(String str) {
```

Comment:
I think `hasText` and `containsText` are redundant, as it completely identical to `isNotBlank`.

Reply:
this named with history spring naming, for less change. It can also be changed to ‘ containsText’